runtime-scientific/fflas-ffpack/autobuild/defines

```diff
@@ -4,3 +4,6 @@ PKGDEP="givaro lapack"
 PKGDES="A library for dense linear algebra over word-size finite fields"
 
 AB_FLAGS_O3=1
+
+AUTOTOOLS_DEF+=(--with-blas-libs="-lcblas -llapack -lgomp")
```

Comment:
Why not set this in `AUTOTOOLS_AFTER`?

Reply:
AUTOTOOLS_AFTER cannot handle `"` properly.

Reply:
`AUTOTOOLS_AFTER="--with-blas-libs=\"-lcblas -llapack -lgomp\""` does not work. `AUTOTOOLS_AFTER="--with-blas-libs=\\\"-lcblas -llapack -lgomp\\\""` neither.

Reply:
Fixed.